src/session/browser.js

```diff
@@ -7,8 +7,8 @@ const {
     SESSION_ID_KEY,
     UTM_CAMPAIGN_KEY,
     MAX_INACTIVITY_TIME,
-} = require("./constants");
-const { UUIDv4, throttle, getItemWithDefault, updateItemWithDefault } = require('./utils');
+} = require("../constants");
```

Comment:
I can revert all the changes done to this file, I end up just replacing this module for QML.

Reply:
You didn't revert :) Should you?